src/content/developers/docs/consensus-mechanisms/pos/gasper/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Gasper
+description: An explanation of the Gasper PoS mechanism.
+lang: en
+sidebar: true
+incomplete: false
+---
+
+Gasper is a combination of Casper the Friendly Finality Gadget and the LMD-GHOST fork choice algorithm. Together these components form the consensus mechanism securing proof-of-stake Ethereum. Casper is the mechanism that uprgades certain blocks to "finalized" so that new entrants into the network can be confident that they are syncing the canonical chain. The fork choice algorithm uses accumulated votes to ensure that when forks arise in the blockchain nodes can easily select the correct one.
+
+**note** that the original definition of Casper-FFG was updated slightly for inclusion in Gasper. On this page we consider the updated version.
+
+## The role of Gasper
```

Comment:
Could you please add header ids to any new headings 🙏
```suggestion
## The role of Gasper {#gasper}
```

Reply:
done